Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "SymbolicUtils"
 uuid = "d1185830-fcd6-423d-90d6-eec64667417b"
 authors = ["Shashi Gowda"]
-version = "2.0.2"
+version = "2.1.0"
```

Comment:
Is this the correct version bump? Or shall this be 3.0.0? Contains breaking

Reply:
what is breaking here?

Reply:
`maketerm` does not take a `symtype` argument anymore because it's inferred from type parameter. `similarterm` was also removed

Reply:
oh yes... I don't have the appetite to do another breaking though. If I make the issue will you setup all of the downstream PRs?

Reply:
Yes. It shouldn't be that hard. I think another breaking is needed then.

Reply:
Let's make sure the sorting change is part of the same change then?

Reply:
> Let's make sure the sorting change is part of the same change then?

The one mentioned in tuesday call, right? Defaulting `arguments` to be unsorted and having `sorted_arguments` instead?